alpha-blog/app/controllers/articles_controller.rb

```diff
@@ -15,6 +15,7 @@ def new
 
   def create
     @article = Article.new(articles_params)
+    @article.user = User.first
```

Comment:
Why `user.first` ?
Why not `@current_user` ?

Reply:
for now only have one user, so use this first, in the next videos should change to `@current_user`